agentarchives/archivesspace/client.py

```diff
@@ -825,7 +825,9 @@ def add_child(self, parent, title="", level="", start_date="", end_date="", date
             if end_date:
                 date['end'] = end_date
 
-        if note and 'content' in note:
+        # If there is a note, but it's an empty string, skip this;
+        # ArchivesSpace doesn't allow subnote content to be empty.
+        if note and note.get('content', ''):
```

Comment:
We probably don't now, but I figured I'd test with the no-key case being identical to the empty-note case for consistency.